### PR TITLE
Add readinessProbe to compose-plugin exposed port

### PR DESCRIPTION
**Pull request by AvoidMe**

### CODE REVIEW

File: modules/dynamic-operator/plugins/docker-compose/plugin/compose/objects.go [hunk 1]
```diff
@@ -345,6 +346,18 @@ func (c *ComposeModel) setApplicationObjects(req *commons.PluginRequest) error {
 				Protocol:      proto,
 			}
 			container.Ports = append(container.Ports, port)
+
+			container.ReadinessProbe = &corev1.Probe{
+				Handler: corev1.Handler{
+					HTTPGet: &corev1.HTTPGetAction{
+						Path: "/healthz",
+						Port: target,
```
Comment (ynnt) [inline modules/dynamic-operator/plugins/docker-compose/plugin/compose/objects.go:354]: Port should be pointing to either container port name (see line 344) or port.ContainerPort (line 345). `target` can be used, but in this case, please make sure in tests that target.String() is pointing to port.Name (which is not) and target.IntVal equals to port.ContainerPort.

Comment (ynnt) [reply modules/dynamic-operator/plugins/docker-compose/plugin/compose/objects.go:354]: Actually, you can transform `target.String() + "-port"` to `target.String()`

**Comment by AvoidMe** [comment]
Ready for review

File: modules/dynamic-operator/plugins/docker-compose/plugin/compose/objects.go (hunk 1 above)
Comment (AvoidMe) [reply modules/dynamic-operator/plugins/docker-compose/plugin/compose/objects.go]: Fixed

Comment (AvoidMe) [reply modules/dynamic-operator/plugins/docker-compose/plugin/compose/objects.go]: I've tried to remove `-port` part, but ci failed instantly and I guess `-port` part actually doing something here, you can check failed test: https://github.com/kuberlogic/kuberlogic/runs/8243755117?check_suite_focus=true

Comment (ynnt) [reply modules/dynamic-operator/plugins/docker-compose/plugin/compose/objects.go]: Fair enough

**Review by telepenin — APPROVED** [review]

**Pull request merged**